### PR TITLE
style(modal): spinner and modal size

### LINKED ISSUE #620: Improve rendering of profile image modal
child of https://github.com/mozilla/fxa-content-server/issues/6261

STEPS TO REPRODUCE
1. Open the profile image modal with or without an profile image set

EXPECTED RESULTS
1. Modal should remain the same size before the image is loaded
2. Spinner is the outline style at 240px square, not the small spinner
3. Default image is SVG that renders well on HiDPI screens

ACTUAL RESULTS
1. Modal goes from small to big as image loads
2. Spinner style is small, not the outline style used on our login page
3. Default image is @1x resolution bitmap

screen shot 2018-06-08 at 8 22 05 am screen shot 2018-06-08 at 8 22 17 am

### PR DESCRIPTION
**Pull request by shreya99oak**
fixes #620 
Pls. review @shane-tomlinson @ryanfeeley

### CODE REVIEW

**Comment by shreya99oak** [comment]
> @shreya99oak pardon the delay, it looks like two of your commits are in this PR, one to clean up app-start and create the clients closer to where they are used, and the other to update the spinner. Can you remove the client related changes?
> 
> We also have a new policy on this repo that all commits must be [GPG signed](https://help.github.com/articles/managing-commit-signature-verification) so that we know the commit came from you and not someone masquerading as you. Can you sign your commit and re-push?

I guess I'll open a new PR with cause this is on the master branch right now

**Comment by shreya99oak** [comment]
> @shreya99oak pardon the delay, it looks like two of your commits are in this PR, one to clean up app-start and create the clients closer to where they are used, and the other to update the spinner. Can you remove the client related changes?
> 
> We also have a new policy on this repo that all commits must be [GPG signed](https://help.github.com/articles/managing-commit-signature-verification) so that we know the commit came from you and not someone masquerading as you. Can you sign your commit and re-push?

Signed it and removed the unnecessary commits. @shane-tomlinson

File: packages/fxa-content-server/app/scripts/templates/settings/avatar_change.mustache [hunk 1]
```diff
@@ -2,7 +2,7 @@
...
+        <img alt="" src="../../../images/spinnerlight.svg">
```
Comment (l-hedgehog) [inline packages/fxa-content-server/app/scripts/templates/settings/avatar_change.mustache]: This change causes FxA to load https://accounts.firefox.com/images/spinnerlight.svg, which returns 404.

Comment (shane-tomlinson) [reply packages/fxa-content-server/app/scripts/templates/settings/avatar_change.mustache]: Aye, you are correct. Opened https://github.com/mozilla/fxa/issues/1315

**Pull request merged**